resources/references/app-reference/flow-action-reference.md

```diff
@@ -0,0 +1,51 @@
+# Flow Action Reference
+
+{% code title="flow-action.xml" %}
+```xml
+<flow-actions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/platform/trunk/src/Core/Framework/App/FlowAction/Schema/flow-action-1.0.xsd">
+    <flow-action>
+        <meta>
+            <name>slack.send.message</name>
```

Comment:
can you add some explanatory comments for the different elements? because right now the reference is not self-explanatory and is hard to understand standalone without the guide